pinot-common/src/main/java/com/linkedin/pinot/common/segment/fetcher/SegmentFetcher.java

```diff
@@ -15,11 +15,10 @@
  */
 package com.linkedin.pinot.common.segment.fetcher;
 
+import java.io.File;
 import java.util.Set;
 import org.apache.commons.configuration.Configuration;
 
-import java.io.File;
```

Comment:
Can you please add some javadocs to the interface methods? If I understand right, we will have more implementations of this interface across file systems, right? It will be really useful to implementors if the expected behavior is documented clearly. Thanks.